src/frontend/contexts/contextExport.js

```diff
@@ -1,3 +1,4 @@
 export { useFilter, FilterProvider } from "./FilterContext";
 export { useNavContext, NavProvider } from "./NavContext";
 export { useProducts, ProductsProvider } from "./ProductContext";
```

Comment:
Name this as index.js, so you don't have to specify filename while importing

Reply:
yup I did that but Its confusing for me, that's why